.circleci/windows/test.bat

```diff
@@ -0,0 +1,70 @@
+@echo on
+@rem Required vars:
+@rem    WIN_OTP_PATH
+@rem    WIN_MSYS2_ROOT
+@rem    ERTS_VERSION
+@rem    PACKAGE_TESTS_DIR
+@rem    PACKAGE_ZIPARCHIVE
+@rem    TEST_STEPS
+@rem    PROJECT_ROOT
+
+SETLOCAL ENABLEEXTENSIONS
+@call:log Set the paths appropriately
+
+:: Appropriately set paths
+SET "PATH=%WIN_MSYS2_ROOT%\mingw64\bin;%WIN_MSYS2_ROOT%\usr\bin;%WIN_MSYS2_ROOT%;%PATH%"
+SET "PATH=%WIN_OTP_PATH%\bin;%WIN_OTP_PATH%\erts-%ERTS_VERSION%\bin;%WIN_OTP_PATH%\erts-%ERTS_VERSION%;%PATH%"
+
+:: Construct unix paths
```

Comment:
Next set of commands will try to resolve where the artifacts are, so this script could be used locally. In addition because msys is a posix environment we need to prepare both versions of windows and unix forms of the paths